README.md

```diff
@@ -1,44 +1,58 @@
 # Mobility Data Specification
 
-A data standard and API specification for *mobility as a service* providers, such as Dockless Bikeshare, E-Scooters, and Shared Ride providers who work within the public right of way.
+The Mobility Data Specification (**MDS**) is a set of data specifications and data sharing requirements focused on dockless e-scooters and bicycles. Inspired by [GTFS](https://developers.google.com/transit/gtfs/reference/) and [GBFS](https://github.com/NABSA/gbfs)m the goals of the MDS are to provide API and data standards for municipalities to help ingest, compare and analyze *mobility as a service* provider data. 
```

Comment:
typo: `[GBFS](https://github.com/NABSA/gbfs)m` (the extra **m** at the end)

Perhaps this is a better place to define API, instead of the section below, since it is used here.